src/plugins/eqht-css/eqht-css.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @title eqht
+ * @overview Provide ability to have equal height containers and nested containers inside a WET-BOEW grid
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @luc-bertrand-hrsdc
+ */
+( function( $, window, wb ) {
+"use strict";
+
+/*
+ * Variable and function definitions.
+ * These are global to the plugin - meaning that they will be initialized once per page,
+ * not once per instance of plugin on the page. So, this is a good place to define
+ * variables that are common to all instances of the plugin on a page.
+ */
+var componentName = "eqht-trgt",
```

Comment:
rename for: `wb-eqht-grd`

Reply:
```suggestion
var componentName = "wb-eqht-grd",
```

Reply:
I could rename it "wb-eqht-trgt".  Renaming it equal height grid would require to change how the whole function works.  Currently the function works by processing from child to parent.  If I rename it to the parent the function will have to work by processing from parent to child.  Let me know.